fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -98,7 +98,25 @@ def suggest_ruby_reinstall(e)
         ui.error "not be available to your system installed Ruby (#{RUBY_VERSION})"
       end
       ui.error ""
-      ui.error "The best solution is to install a new version of Ruby"
+      ui.error "The best solution is to use the self-contained fastlane version."
+      ui.error "Which ships with a bundled OpenSSL,ruby and all gems - so you don't depend on system libraries"
+      ui.error " - Use One-Click-Installer:"
+      ui.error "    - download fastlane at https://download.fastlane.tools"
+      ui.error "-----------------------------------------------------------"
+      ui.error "    - extract the archive and double click the `install`"
+      ui.error "-----------------------------------------------------------"
+      ui.error " - Use Homebrew"
+      ui.error "    - if you do not have brew-cask enabled run: `brew tap caskroom/cask`"
```

Comment:
Hmm - that error message sounds more like they need to run `brew update`. Checking `homebrew-cask`'s official docs they also don't mention having to tap the Caskroom anymore: https://github.com/caskroom/homebrew-cask/blob/master/USAGE.md